flask_caching/__init__.py

```diff
@@ -132,6 +131,20 @@ def function_namespace(f, args=None):
     return ns, ins
 
 
+def to_str(str_or_byte):
+    if isinstance(str_or_byte, bytes):
+        return str_or_byte.decode("utf8")
```

Comment:
This can lead to similar problems as the original `str(x)` approach. If `str_or_bytes` contains non-UTF8 byte sequences, you will get an exception.

Reply:
yep. old code has the same prolem。
i think， this function should have the argument of character encoding。